src/main/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFs.java

```diff
@@ -148,6 +155,12 @@
     /** these are used when {@link #onlyOutputCallsStartingInIntervals) is true */
     private List<SimpleInterval> intervals;
 
+
+    @Override
+    protected int getVariantCacheLookAheadBases() {
+        return 1000;
```

Comment:
Do heavy-duty profiling to determine the effect on both the memory use and runtime of `GenotypeGVCFs`.
(and bear in mind that for the 20k gnomAD run, it was the native heap memory use by GenomicsDB that was a problem (10+ GB in some cases), whereas the Java heap usage stayed within a fixed and relatively low `-Xmx` value (4-5 GB))